Automattic/pocket-casts-ios
# [Global Playback Settings]  Enable Feature Flag

| 📘 Part of: #2253
|:---:|

Fixes #2297

This PR enable the FF and update the CHANGELOG

## To test

- CI must be 🟢 
- Make sure the FF is on and you see the settings into the Effects sheet

## Checklist
